File: Embedded C/04_Bitwise Operators/Worksheet_2/5.c
#include <stdio.h>

#define CLEAR_BIT(num, n) ((num) & ~(1U << (n)))

int main() {
    unsigned int num;
    int bit_pos = 0;

    printf("Input number: ");
    scanf("%u", &num);
    
    printf("%d -> %b\n", num, num);
    unsigned int bit_value = CLEAR_BIT(num, bit_pos);

    printf("Set %dth bit of %u is %b\n", bit_pos, num, bit_value);

    return 0;
}

File: Python/04 Loops/5.py
num = int(input("Enter an integer: "))
if num == 0:
    print(0)
else:
    if num < 0:
        print("-", end="")
        num = -num
    while num > 0:
        print(num % 10, end="")
        num //= 10
    print()